packages/react-interactions/events/src/dom/Press.js

```diff
@@ -106,7 +106,10 @@ function isValidKey(e): boolean {
 
 function handlePreventDefault(preventDefault: ?boolean, e: any): void {
   const key = e.key;
-  if (preventDefault !== false && (key === ' ' || key === 'Enter')) {
+  if (
+    preventDefault !== false &&
+    (key === ' ' || key === 'Enter' || key === 'Spacebar')
```

Comment:
Why is `Spacebar` added here? For some legacy browser? It would need to be added to L100 too.

Reply:
For legacy browser support. I've added to the other code-site as per your suggestion.